airflow/hooks/base_hook.py

```diff
@@ -88,6 +88,20 @@ def get_hook(cls, conn_id):
         connection = cls.get_connection(conn_id)
         return connection.get_hook()
 
+    @classmethod
```

Comment:
Could you provide some detail about this method and the implementation? It's a bit confusing to me.

I.e. Pydoc describing what input/output to expect and the behaviour of the method.